### ISSUE TITLE
virtualenv installation method can result in "no such file or directory" error, if working on a directory with space characters

### ISSUE THREAD

**Issue by mnahinkhan** [mailpile/Mailpile#2276]
I am working from a terminal on Mac. I run the following after cloning the project:
```
$ make virtualenv
$ source mp-virtualenv/bin/activate
$ mailpile
```

This results in an error if the directory one is working in involves spaces. ("No such file or directory: " followed by a prefix of the directory without spaces).

Thankfully, there is a simple fix; before calling the "mailpile" command I went to ./mp-virtualenv/bin/python directory, and edited the 'mailpile' file's first line as follows:
Before:
```
#!/.../ dir with spaces/.../Mailpile/mp-virtualenv/bin/python
```
After:
```
#'!/.../ dir with spaces/.../Mailpile/mp-virtualenv/bin/python'
```

In other words, I just had to add the single quotation marks around it and everything worked fine after. Unfortunately, I wasn't too sure which part of the code auto-generated this file (I traced it down to a "python install setup.py" command, but not sure beyond that), but wherever it is getting autogenerated, it might help to put the quotation marks around the python directed directory.

Let me know if you need more details!
